tests/integration/examples/utils.py

```diff
@@ -141,19 +149,17 @@ def run_example(
             # Docker is not installed or running
             pass
         else:
-            try:
-                # If the image is not present on the local machine, the build
-                # phase either failed or we don't even have an orchestrator that
-                # builds a container image. Either way, we don't need to clean
-                # up any Docker resources.
-                pipeline_image = docker_client.images.get(image_name)
-                docker_client.containers.prune()
-                docker_client.volumes.prune()
-                logging.debug(f"Removing Docker image {pipeline_name}")
-                docker_client.images.remove(pipeline_image.id)
-                docker_client.images.prune()
-            except ImageNotFound:
-                pass
+            docker_client.containers.prune()
+            docker_client.volumes.prune()
+            for image_name in image_names:
+                try:
+                    logging.debug(f"Removing Docker image {image_name}")
+                    image = docker_client.images.get(image_name)
+                    docker_client.images.remove(image.id, force=True)
```

Comment:
Why use the force here ? Assuming there is no container using the image, the image should want to die voluntarily.

Reply:
If the same image ID is tagged twice (e.g. the training and inference pipeline of the quickstart), Docker refuses to delete the image even if no container is using it